docs/settings/logging-settings.asciidoc

```diff
@@ -4,6 +4,18 @@
 <titleabbrev>Logging settings</titleabbrev>
 ++++
 
+{kib} relies on three high-level entities to set the logging service: _appenders_, _loggers_ and _root_.
```

Comment:
Consider putting the three items in a bulleted list instead of separate paragraphs:

- Appenders define....
- Loggers define ...
- Root is a special

To improve readability, consider removing the italics and parens.

Reply:
Done! Thank you!